src/sample.appsettings.json

```diff
@@ -3,7 +3,8 @@
     "Path": "/api/health",
     "IncludeDbContext": [ "MyDBContextName1", "MyDBContextName2"],
     "ResourceGroupName": "NameOfResourceGroup",
-    "SubscriptionId": "IDofSubscription", 
+    "SubscriptionId": "IDofSubscription",
+    "AppInsightsInstrumentKey": "",
```

Comment:
AppInsightsInstrument**ation**Key

Reply:
I'll update it when I do a PR for the documentation around using the publisher.